### PR TITLE
Unify tools and clean up

### PR DESCRIPTION
**Pull request by wwylele**
Now all scripts runs purely on python3+sh. Removed 3dstool dependency.

### CODE REVIEW

**Review by B3n30 — APPROVED** [review]
LGTM as soon as travis finishes

**Comment by B3n30** [comment]
`No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.` seems like you need to add some debug logs :-P

File: common/lz_compress.py [hunk 1]
```diff
@@ -131,10 +131,16 @@ def _compress(input):
...
 
+    percentage = 0
+
     i = 0
     while True:
         if len(input) <= i:
             break
+        new_percentage = int(i / len(input) * 100)
+        if percentage != new_percentage:
+            print("{}%".format(new_percentage))
```
Comment (B3n30) [inline common/lz_compress.py:144]: just ultimately nitpicking here: `print("{}%\r".format(new_percentage), end="")` ? ;-)

**Review by B3n30 — APPROVED** [review]
no need to change the nitpicking I mentioned if you don't won't to. 

LGTM

File: common/lz_compress.py (hunk 1 above)
Comment (wwylele) [reply common/lz_compress.py:144]: Couldn't get this working with Travis

**Pull request merged**